CHANGELOG.md

```diff
@@ -21,17 +31,16 @@
 
 **_Thank You_**
 
-* [@blindpirate (Bo Zhang)](https://github.com/blindpirate): Show pull request's close button for author [PR #3507](https://github.com/microsoft/vscode-pull-request-github/pull/3507)
-* [@leopoldsedev (Christian Leopoldseder)](https://github.com/leopoldsedev): Implement quick self assign link as available on .com (#3382) [PR #3601](https://github.com/microsoft/vscode-pull-request-github/pull/3601)
-
+- [@blindpirate (Bo Zhang)](https://github.com/blindpirate): Show pull request's close button for author [PR #3507](https://github.com/microsoft/vscode-pull-request-github/pull/3507)
```

Comment:
Lots of surprising formatting changes here. Is this coming from prettier and if so, can they be reverted?

Reply:
Formatter seems to have run when I saved the file. I'll undo all this weird stuff.

Reply:
I use `"editor.formatOnSaveMode": "modificationsIfAvailable"` to prevent this. If I need to format the whole document then I can always run the Format Document command later.